### PR TITLE
Menu accessibility tab doc update

### PR DESCRIPTION
**Pull request by alina-jacob**
Closes https://github.com/carbon-design-system/carbon-website/issues/3796

**New**

- [Accessibility Tab] Added content and images for Menu.

**Changed**

- [Usage tab] Updated guidance and image for Overflow content (truncated menu item text) .
- [Usage tab] Updated the RTL image to show the right chevron and command directions.

### CODE REVIEW

File: src/pages/components/menu/accessibility.mdx [hunk 1]
```diff
@@ -6,10 +6,80 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
+Carbon bakes keyboard operation into its components, improving the experience of
+blind users and others who operate via keyboard. Carbon also incorporates other
+accessibility considerations, some of which are described below.
+
+### Keyboard interaction
+
+The menu is reached by tab. When the menu is open, the first item takes focus.
+The menu items are navigated using the `Up` and `Down` arrow keys.
+
+Row>
```
Comment (ariellalgilmore) [inline src/pages/components/menu/accessibility.mdx:34]:
```suggestion
<Row>
```

**Review by alisonjoseph — APPROVED** [review]

File: src/pages/components/menu/accessibility.mdx [hunk 2]
```diff
@@ -6,10 +6,80 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
+
+## What Carbon provides
+
+Carbon bakes keyboard operation into its components, improving the experience of
+blind users and others who operate via keyboard. Carbon also incorporates other
+accessibility considerations, some of which are described below.
+
+### Keyboard interaction
+
+The menu is reached by tab. When the menu is open, the first item takes focus.
```
Comment (laurenmrice) [inline src/pages/components/menu/accessibility.mdx:31]:
```suggestion
The menu is reached by `Tab`. When the menu is open, the first item takes focus.
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looking good Alina! Left a couple comments below:

**Review by Kritvi-bhatia17 — APPROVED** [review]
Great work Alina! LGTM 🚀

If you have any research or iterations documented in a Figma file or Box note, could you please share them in the [issue's comments](https://github.com/carbon-design-system/carbon-website/issues/3796) (like this:)

image

File: src/pages/components/menu/accessibility.mdx [hunk 3]
```diff
@@ -6,10 +6,83 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
+  Menu is reached by Tab and when opened, the first menu item takes focus. Arrow
+  keys move focus.
+</Caption>
+
+</Column>
+</Row>
+
+Pressing `Space` or `Enter` on a focused menu item activates it and collapses the 
+menu. Pressing `Space` or `Enter` on a submenu opens it. A submenu can also be 
+opened and closed using the right and left arrow key respectively. When the menu or
```
Comment (laurenmrice) [inline src/pages/components/menu/accessibility.mdx:50]:
```suggestion
opened and closed using the right and left arrow keys respectively. When the menu or 
```

File: src/pages/components/menu/accessibility.mdx [hunk 4]
```diff
@@ -6,10 +6,83 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
+opened and closed using the right and left arrow key respectively. When the menu or 
+submenu is open, the first item takes focus. An activated menu item can have 
+different results depending on its functionality. If the item is a link, it launches
+the selection. If the item is selectable, it selects or deselects the item. The menu 
+item retains its selected or unselected state when the menu is opened again.
+
+<Row>
+<Column colLg={8}>
+
+![example of menu keyboard interaction](images/accessibility-menu-2.png)
```
Comment (laurenmrice) [inline src/pages/components/menu/accessibility.mdx:59]: I have left a comment in your Figma file about this image.

**Review by laurenmrice — APPROVED** [review]

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**